modules/registry-operator-default-crd.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * assembly/registry
+// * openshift_images/configuring-registry-operator.adoc
+
+
+[id="registry-operator-default-crd-{context}"]
+= Image Registry Operator Custom Resource Definition and parameter values
```

Comment:
Is this a procedure or a reference? I don't see the default CRD. If it's a procedure, you need a different title.